autoemulate/experimental/data/utils.py

```diff
@@ -93,6 +93,18 @@ def _convert_to_tensors(
             f"Unsupported type for dataset ({type(dataset)}). Must be TensorDataset."
         )
 
+    def _convert_to_numpy(
+        self,
+        x: InputLike,
+        y: InputLike | None = None,
+    ) -> tuple[np.ndarray, np.ndarray]:
+        """
+        Convert InputLike x, y to tuple of numpy arrays.
+        """
+        x, y = self._convert_to_tensors(x, y)
```

Comment:
if y is None, `_convert_to_tensors` returns just an x so we need to handle that in the code